lib/MCSResponseClient.js

```diff
@@ -507,6 +509,31 @@ class MCSResponseClient extends MCSBaseClient {
     }
   }
 
+  conferenceFloor(roomId, mediaId) {
```

Comment:
For both conferenceFloor and contentFloor: if you don't pass an extra parameter with the transaction ID, the API call won't be deferred and so treating it as a promise won't work (e.g.: async/await). Please fix this, you can use the `subscribe/subscribed` messages as an example. Here:
https://github.com/mconftec/mcs-js/blob/mconf/lib/MCSResponseClient.js#L366
https://github.com/mconftec/mcs-js/blob/mconf/lib/messages/subscribed.js#L7-L8